mgmt/rest/v1/fixtures/mock_task_manager.go

```diff
@@ -249,7 +250,7 @@ const (
       }
     },
     "schedule": {
-      "type": "simple",
+      "type": "windowed",
```

Comment:
After including "simple" into "windowed" schedule, the getting task response will contain "windowed" as schedule type